### PR TITLE
Update `policy-framework` docs for `clusteradm`

### PR DESCRIPTION
**Pull request by dhaiducek**
Depends on:
- https://github.com/open-cluster-management-io/clusteradm/pull/229

Addresses:
- https://github.com/stolostron/backlog/issues/16046
- https://github.com/open-cluster-management-io/community/issues/122

### CODE REVIEW

File: content/en/getting-started/integration/policy-controllers.md [hunk 1]
```diff
@@ -24,9 +24,32 @@ You must meet the following prerequisites to install the policy controllers:
 - Ensure the `open-cluster-management` _policy framework_ is installed. See
   [Policy Framework](/getting-started/integration/policy-framework) for more information.
 
-## Install the configuration policy controller
+## Install the configuration policy controller to the managed cluster(s)
 
-Complete the following procedure to install the configuration policy controller:
+### Deploy via Clusteradm CLI
+
+Ensure `clusteradm` CLI is installed and is newer than v0.2.0. Download and extract the
```
Comment (JustinKuli) [inline content/en/getting-started/integration/policy-controllers.md:31]: Newer than? Or at least? What version is required?

File: content/en/getting-started/integration/policy-controllers.md [hunk 2]
```diff
@@ -24,9 +24,32 @@ You must meet the following prerequisites to install the policy controllers:
...
+[clusteradm binary](https://github.com/open-cluster-management-io/clusteradm/releases/latest). For
+more details see the
+[clusteradm GitHub page](https://github.com/open-cluster-management-io/clusteradm/blob/main/README.md#quick-start).
+
+1. Deploy the configuration policy controller to the managed cluster(s) (this command is the same
+   for a managed hub):
```
Comment (JustinKuli) [inline content/en/getting-started/integration/policy-controllers.md:37]: Do you mean a self-managed hub?

File: content/en/getting-started/integration/policy-controllers.md [hunk 3]
```diff
@@ -24,9 +24,32 @@ You must meet the following prerequisites to install the policy controllers:
...
+
+   ```Shell
+   # Deploy the configuration policy controller
+   clusteradm addon enable addon --names configuration-policy-controller --clusters <cluster_name> --context ${CTX_MANAGED_CLUSTER}
```
Comment (JustinKuli) [inline content/en/getting-started/integration/policy-controllers.md:41]: Is it confusing that the addon is named `configuration-policy-controller` here, but the actual deployment on the managed cluster, and the ManagedClusterAddon are both `config-policy-controller`?

File: content/en/getting-started/integration/policy-framework.md [hunk 4]
```diff
@@ -228,74 +229,109 @@ Ensure `clusteradm` CLI is installed and is newer than v0.2.0. Download and extr
...
+2. Verify that the pod is running on the managed cluster with the following command:
 
-## Deploy the synchronization components to the managed cluster(s)
+   ```Shell
+   $ kubectl get pods -n open-cluster-management-agent-addon
+   NAME                                               READY   STATUS    RESTARTS   AGE
+   governance-policy-framework-57579b7c-652zj         3/3     Running   0          87s
+   ```
+
+   **NOTE**: A managed hub deployment will only have 2 containers in the pod as opposed to the `3/3`
```
Comment (JustinKuli) [inline content/en/getting-started/integration/policy-framework.md:326]: Again, should this be "self-managed"?

File: content/en/getting-started/integration/policy-controllers.md (hunk 3 above)
Comment (dhaiducek) [reply content/en/getting-started/integration/policy-controllers.md:41]: Good catch. These docs are wrong--it's called `config-policy-controller`.

File: content/en/getting-started/integration/policy-controllers.md (hunk 1 above)
Comment (dhaiducek) [reply content/en/getting-started/integration/policy-controllers.md:31]: Good point. Actually, by the time the changes are released, it'll probably need to be "at least `v0.3.0`", so we can't really merge this PR until the changes are in AND a new release is created for `clusteradm`.

**Comment by mikeshng** [comment]
Please feel free to make a separate PR if you think a member from the policy-sig should be added to the https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/blob/main/OWNERS file so he/she can approve policy related changes. CC @mprahl @dhaiducek @gparvin

**Review by JustinKuli — APPROVED** [review]
LGTM! Of course, needs https://github.com/open-cluster-management-io/clusteradm/pull/229 first, but that's looking almost ready.

/hold

File: content/en/getting-started/integration/policy-controllers.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhaiducek) [reply content/en/getting-started/integration/policy-controllers.md:31]: Good catch!

**Review by mprahl — APPROVED** [review]
Nice!

**Comment by dhaiducek** [comment]
(This PR needs to continue to hold for the `v0.3.0` release of `clusteradm`.)

**Pull request merged**